ui/pages/confirmations/hooks/useCurrentConfirmation.ts

```diff
@@ -34,22 +34,27 @@ const useCurrentConfirmation = () => {
       pendingConfirmation = pendingConfirmations.find(
         ({ id: confirmId }) => confirmId === paramsTransactionId,
       );
-    } else {
+    }
+    if (!pendingConfirmation && latestPendingConfirmation) {
       pendingConfirmation = latestPendingConfirmation;
     }
-    if (
-      pendingConfirmation &&
-      pendingConfirmation.id !== currentConfirmation?.id
-    ) {
+    if (!pendingConfirmation) {
+      setCurrentConfirmation(undefined);
+      return;
+    }
```

Comment:
What do you feel about the change below?
```js
if (!pendingConfirmation){
  if (!latestPendingConfirmation) {
    setCurrentConfirmation(undefined);
    return;
  }

  pendingConfirmation = latestPendingConfirmation;
}
```

This change would reduce the cyclomatic complexity from 3 to 2.

Reply:
looks good, let me make this change